extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -615,11 +629,10 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.polyOptions.bboxPoints.push(
```

Comment:
Explain what startPt is and how it is determined.

Reply:
Done.